netbox/extras/models/customfields.py

```diff
@@ -275,14 +275,16 @@ def clean(self):
             try:
                 if self.type in (CustomFieldTypeChoices.TYPE_TEXT, CustomFieldTypeChoices.TYPE_LONGTEXT):
                     default_value = str(self.default)
+                elif self.type in (CustomFieldTypeChoices.TYPE_MULTISELECT, CustomFieldTypeChoices.TYPE_MULTIOBJECT):
+                    default_value = [self.default]
```

Comment:
The default value should be provided in this form; i.e. `"test"` is not a valid default value for a multi-select field. The correct default value would be `["test"]`, because that's the native form of the data.